src/components/ModalWindow/__spec__/modal.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
why did you call file modal.*?

Reply:
becouse it shorter then "modalWindow"
renamed fale

Reply:
fileName of the test should be same as components file name